src/symmetric.jl

```diff
@@ -848,7 +854,8 @@ function ^(A::SelfAdjoint, p::Real)
     isinteger(p) && return integerpow(A, p)
     F = eigen(A)
     if all(λ -> λ ≥ 0, F.values)
-        retmat = (F.vectors * Diagonal((F.values).^p)) * F.vectors'
+        map!(λ -> λ^0.5p, F.values)
```

Comment:
```suggestion
        map!(λ -> λ^0.5p, F.values, F.values)
```
The in-place `map!` requires the destination as well.

Reply:
Oh. I see that there's a 2-term method as well. Strangely, this is not extended by `StaticArrays`.

Reply:
It doesn't, this changed in 1.12.